ecosystem/indexer/src/models/events.rs

```diff
@@ -23,12 +23,27 @@ pub struct Event {
 
 impl Event {
     pub fn from_event(transaction_hash: String, event: &APIEvent) -> Self {
+        let mut _data = event.data.clone();
+        let from_message = serde_json::to_string(&_data["from_message"]).unwrap();
+        let to_message = serde_json::to_string(&_data["from_message"]).unwrap();
+
+        _data["from_message"] = serde_json::Value::from(
+            from_message
+                .trim_matches(char::from(0))
+                .replace("\\u0000", ""),
+        );
+        _data["to_message"] = serde_json::Value::from(
+            to_message
+                .trim_matches(char::from(0))
+                .replace("\\u0000", ""),
+        );
+
```

Comment:
more specifically, this could appear anywhere in the message: maybe we want a helper that replaces null bytes in any string or json field?.